### PR TITLE
Added Single Command for Frontend + Backend

### LINKED ISSUE #3904: Add a single command to start server and frontend
Saw this in /activepieces Github project:

```
> cross-env-shell FORCE_COLOR=true concurrently -n "GUI,API,ENG,PCS" -c "bgBlue.bold,bgGreen.bold,bgRed.bold,bgYellow.bold" "npm:serve:frontend" "npm:serve:backend" "npm:serve:engine" "npm:serve:pieces"
```

Screenshot 2024-02-09 at 18 07 45

I feel like it could be handy to have a single command to run backend/frontend at the same time too? 
We could call it `yarn start` (add to package.json)

### PR DESCRIPTION
**Pull request by Kanav-Arora**
Closes #3904 

Screenshot 2024-02-10 at 10 32 01 PM

### CODE REVIEW

**Comment by Kanav-Arora** [comment]
@FelixMalfait ready

**Review by charlesBochet — APPROVED** [review]
LGTM, thank you @Kanav-Arora! that's improving the DevXP!

Added minor comment

File: package.json [hunk 1]
```diff
@@ -279,6 +280,9 @@
     "graphql": "16.8.0"
   },
   "version": "0.2.1",
+  "scripts": {
+    "start": "cross-env FORCE_COLOR=true concurrently -n \"TwentyServer,TwentyFrontend\" -c \"bgBlue.bold,bgGreen.bold\" \"yarn nx start:dev twenty-server\" \"yarn nx start twenty-front\""
```
Comment (charlesBochet) [inline package.json:284]: Can we replace "TwentyServer,TwentyFrontend" by "twenty-server,twenty-front" to be consistent folder cases.

Let's give it a week to try it on various environments and then we can add it to the docs
Also, could you merge main into your branch, tests should be green!

**Comment by charlesBochet** [comment]
@Kanav-Arora works like a charm, thank you!

File: package.json (hunk 1 above)
Comment (Kanav-Arora) [reply package.json]: Hi @charlesBochet 
Sorry I just saw this I was bit busy

**Pull request merged**